atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueCache.java

```diff
@@ -31,10 +32,16 @@
  * the central cache (taking in to account the since-locked descriptors), as well as checking for conflicts for
  * serializable transactions by adding a check in the {@link com.palantir.atlasdb.transaction.api.PreCommitCondition}.
  */
-public interface LockWatchValueCache {
+public interface LockWatchValueCache extends ValueCacheUpdater {
+    @Override
     void processStartTransactions(Set<Long> startTimestamps);
 
-    void updateCacheOnCommit(ValueDigest digest, long startTs);
+    @Override
+    void updateCacheOnCommit(Set<Long> startTimestamps);
```

Comment:
You can merge in https://github.com/palantir/atlasdb/pull/5408/files for this change (ideally, we get that merged before this).